### PR TITLE
Feat: Adds `Toggle` & `ToggleField` & `SROnly` components

### PR DESCRIPTION
**Pull request by hellofanny**
## What's the purpose of this pull request?

- Creates `Toggle` component on @faststore/components
- Adds `Toggle` styles on @faststore/ui
- Adds `Toggle` documentation page on Docusaurus
- Creates `ToggleField` component on @faststore/components
- Adds `ToggleField` styles on @faststore/ui
- Adds `ToggleField` documentation page on Docusaurus
- Creates `SROnly` component on @faststore/components
- Adds `SROnly` styles on @faststore/ui
- Adds `SROnly` documentation page on Docusaurus

## How it works?

`Toggle` -> Custom Input type checkbox (without a label) looking like a toggle
`ToggleField` -> Group a Toggle input and its corresponding Label.
`SROnly` -> Hides an element visually. Improve accessibility by providing text to screen readers when applied.

## How to test it?

run `yarn start` in app/docs folder and checks the Toggle, SROnly component or test it using the doc [preview link.](https://faststore-git-feat-toggle-component-fs-614-faststore.vercel.app/reference/ui/molecules/Toggle)

### Starters Deploy Preview

Feat: Replaces SROnly & Toggle & ToggleField FSUI: https://github.com/vtex-sites/nextjs.store/pull/318

### CODE REVIEW

File: apps/docs/docs/reference/ui/atoms/SROnly.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+description: Hides an element visually. Improve accessibility by providing text to screen readers when applied.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Badge___bcedffeb307527c1606be224a7fb5ac1.png
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/atoms/SROnly.mdx:8]: @carolinamenezes  We need to update this image. 🙈

File: apps/docs/docs/reference/ui/molecules/ToggleField.mdx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+---
+description: ToggleField group a Toggle input and its corresponding Label.
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/RadioLabel___b3fc3b7a139796e4921349df5aaa270c.png
```
Comment (hellofanny) [inline apps/docs/docs/reference/ui/molecules/ToggleField.mdx:4]: @carolinamenezes  We need to update this image. 🙈

**Review by lucasfp13 — APPROVED** [review]
Nice, Fan! 💯

File: packages/components/src/molecules/ToggleField/ToggleField.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import { Label, SROnly, Toggle } from './../../'
+
+export type ToggleFieldProps = {
```
Comment (renatamottam) [inline packages/components/src/molecules/ToggleField/ToggleField.tsx:4]: Missing testID here too?

File: packages/components/src/molecules/Toggle/Toggle.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+   */
+  disabled?: boolean
+  /**
+   * Controls the component's direction.
+   */
+  variant?: 'horizontal' | 'vertical'
+}
+
+const Toggle = forwardRef<HTMLInputElement, ToggleProps>(function Toggle(
+  { id, disabled, variant = 'horizontal', ...otherProps }: ToggleProps,
```
[the remark this answers is not in the document]
Comment (hellofanny) [reply packages/components/src/molecules/Toggle/Toggle.tsx:23]: I forgot to add it! I will add it! Thankssssss 🚀

File: apps/docs/docs/reference/ui/molecules/Toggle.mdx [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+# Toggle
+
+The Toggle is a customized checkbox input styled to look like a switch button.
+
+## Overview
+
+```tsx live
+<div className="overviewSection">
+  <Toggle id="toggle-default" />
+  <Toggle id="toggle-default-labeless" />
```
[the remark this answers is not in the document]
Comment (hellofanny) [reply apps/docs/docs/reference/ui/molecules/Toggle.mdx:19]: nops! Removed.

**Review by emersonlaurentino — APPROVED** [review]

**Review by renatamottam — APPROVED** [review]
Perfect!

**Pull request merged**